### DISCUSSION TITLE
Customizing Output Parameters

### DISCUSSION THREAD

**Discussion by ehsanj25** [NCAR/hrldas#73] [Q&A] [answered]
Domain setup and namelist, etc.

Hello everyone,
I have a question regarding the configuration options for selecting specific output parameters. I noticed that there is an option to select NoahMP output level (NOAHMP_OUTPUT) where 0 represents the standard parameters and 1 represents the standard parameters along with the additional ones. However, I would like to have the ability to specifically select the parameters that I need while excluding any unnecessary ones to reduce the output file size, is there any way to achieve this customization?

**Comment by cenlinhe** [comment, marked as answer]
Hi, thank you for your question. Currently we do not have such capabilities through namelist control (we hope to add this capability in the future). If you would like to select specific output variables for now, you need to go to the [hrldas/hrldas/IO_code/](https://github.com/NCAR/hrldas/blob/master/hrldas/IO_code/module_NoahMP_hrldas_driver.F#L1093), and comment out variables you do not need to output and uncomment variables you would like to output.

**Reply by SmithSimmon** [reply]
Hi, @cenlinhe, I met the same issue. I commented out some variables that I don't need to output in _hrldas/hrldas/IO_code/module_Noahmp_hrldas_driver.f90_, as shown in the picture below. Then I ran the ''make'' command in the _hrldas/hrldas_ diectory. However, when I executed the ./hrldas.exe, the output file still includes the same number of output variables(like standard situation).  is there anything wrong with the steps I followed?
<img width="632" alt="bfa3d83590526e998a34279a458acd0" src="https://github.com/user-attachments/assets/26474e69-893e-485b-ad48-451a54eda850">

**Reply by cenlinhe** [reply]
you will need to ran "make clean" first and then "make" to re-compile the code.

**Reply by SmithSimmon** [reply]
Thanks so much! Now I have the variables I need.
